bundles.go

```diff
@@ -66,7 +66,7 @@ func getBundles(ctx context.Context, r *Runtime) ([]*Bundle, error) {
 	for _, rs := range queryResults.Result {
 		v, ok := rs.Bindings["x"].(string)
 		if !ok {
-			r.Logger.Error().Err(fmt.Errorf("expected binding [x] not found")) //nolint:goerr113
+			r.Logger.Error().Err(errors.New("expected binding [x] not found"))
```

Comment:
This looks really strange. Why not write it as:
```go
r.Logger.Error().Msg("expected binding [x] not found")
```

Reply:
Agreed, fixed